zkvm/Cargo.toml

```diff
@@ -10,17 +10,7 @@ itertools = "0.12.0"
 # serialization
 log = "0.4.0"
 
-#powdr-backend =        { git = "https://github.com/powdr-labs/powdr", rev = "97ea8d0" }
-#powdr-pipeline =       { git = "https://github.com/powdr-labs/powdr", rev = "97ea8d0" }
-#powdr-riscv =          { git = "https://github.com/powdr-labs/powdr", rev = "97ea8d0" }
-#powdr-riscv-executor = { git = "https://github.com/powdr-labs/powdr", rev = "97ea8d0" }
-#powdr-number =         { git = "https://github.com/powdr-labs/powdr", rev = "97ea8d0" }
-
-powdr-backend =        { git = "https://github.com/eigmax/powdr", branch = "main" }
-powdr-pipeline =       { git = "https://github.com/eigmax/powdr", branch = "main" }
-powdr-riscv =          { git = "https://github.com/eigmax/powdr", branch = "main" }
-powdr-riscv-executor = { git = "https://github.com/eigmax/powdr", branch = "main" }
-powdr-number =         { git = "https://github.com/eigmax/powdr", branch = "main" }
+powdr = { git = "https://github.com/powdr-labs/powdr", rev = "af97517" }
```

Comment:
Is it possible for us to use main branch one day?

Reply:
The `main` branch would work right now already, this hash was the same as main at the time of writing. But I don't think it makes sense because it will break things on your side without any warning when we merge breaking changes to `main`. I think you should pin to a specific commit (or tag) and update that once in a while, like we do for `starky` in powdr.

Reply:
I have updated the fork to the latest main of upstream, https://github.com/eigmax/powdr can you help change the repo back to eigmax? so we can keep testing this PR and merge it?